Add ENV var to control autoKiller for seperate process on the HTML validator (#601)
rooseveltframework/roosevelt: README.md

- `ROOSEVELT_VALIDATOR`:
  - Set to `detached` to force the HTML validator to run as a detached background process.
  - Set to `attached` to force the HTML validator to run as an attached process.
- `ROOSEVELT_AUTOKILLER`:

>   - Set to `on` to turn on the HTML validator autokiller on the separate process.

Reviewer: ```suggestion
  - Set to `on` to spawn a process to kill the HTML validator if it is running in the background and idle for more than a certain amount of time. The timeout can be configured in [app behavior params](https://github.com/rooseveltframework/roosevelt#app-behavior-parameters).
```

>   - Set to `off`to turn off the HTML validator autokiller on the separe process.

Reviewer: ```suggestion
  - Set to `off`to turn off the HTML validator autokiller on the separate process.
```

>   - Set to `off`to turn off the HTML validator autokiller on the separate process.

Reviewer: ```suggestion
  - Set to `off`to disable the HTML validator autokiller.
```
